recipes/docker.rb

```diff
@@ -4,6 +4,8 @@
 #
 # Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
 
+node.override['nexus_iq_server']['logging']['archived_file_count'] = 5
```

Comment:
Does this need to be override? Can it just be `default['nexus_iq_server']['logging']['archived_file_count'] = 5`?

Reply:
Changed in favor of your excellent suggestion:

```
node.default['nexus_iq_server']['config']['server']['requestLog']['appenders'][0]['archivedFileCount'] = 5
node.default['nexus_iq_server']['config']['logging']['appenders'][1]['archivedFileCount'] = 5
```
(2cdb85f)